app/src/main/res/layout/fragment_breathless.xml

```diff
@@ -82,16 +82,9 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toBottomOf="@id/cause_recycler_view" />
 
-        <ProgressBar
-            android:id="@+id/progress"
-            style="@style/screen_progress_indicator"
-            isVisible="@{vm.isInProgress}"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent" />
+        <include
+            layout="@layout/progress_bar_layout"
+            isVisible="@{vm.isInProgress}"/>
```

Comment:
You still need here the constraints to pin to the edges. These are context-dependent (by having the `include` inside a `ConstraitLayout`), so should be here.